### PR TITLE
Lit Dao Governance Strategy [lit-dao-governance]

### PR DESCRIPTION
**Pull request by wizardlabsxyz**
This is a custom voting strategy for LIT DAO

### CODE REVIEW

File: src/strategies/lit-dao-governance/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# erc721
```
Comment (ChaituVR) [inline src/strategies/lit-dao-governance/README.md:1]:
```suggestion
# lit-dao-governance
```

File: src/strategies/lit-dao-governance/index.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { strategy as masterchefStrategy } from '../masterchef-pool-balance';
+
+export const author = 'wizardlabs';
```
Comment (ChaituVR) [inline src/strategies/lit-dao-governance/index.ts:5]:
```suggestion
export const author = 'wizardlabsxyz';
```

File: src/strategies/lit-dao-governance/schema.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          "examples": ["e.g. DOODLE"],
+          "maxLength": 16
+        },
+        "address": {
+          "type": "string",
+          "title": "Contract address",
+          "examples": ["e.g. 0x1f9840a85d5af5bf1d1762f925bdaddc4201f984"],
+          "pattern": "^0x[a-fA-F0-9]{40}$",
+          "minLength": 42,
+          "maxLength": 42
```
Comment (ChaituVR) [inline src/strategies/lit-dao-governance/schema.json:21]: This is not needed if params are not passed, maybe just allow `Symbol`

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/lit-dao-governance/README.md (hunk 1 above)
Comment (wizardlabsxyz) [reply src/strategies/lit-dao-governance/README.md]: fixed

File: src/strategies/lit-dao-governance/index.ts (hunk 2 above)
Comment (wizardlabsxyz) [reply src/strategies/lit-dao-governance/index.ts]: fixed

File: src/strategies/lit-dao-governance/schema.json (hunk 3 above)
Comment (wizardlabsxyz) [reply src/strategies/lit-dao-governance/schema.json]: fixed

**Comment by wizardlabsxyz** [comment]
all comments, resolved with latest.

**Review by ChaituVR — APPROVED** [review]

**Review by ChaituVR — CHANGES_REQUESTED** [review]
@wizardlabsxyz code looks good but there are few tests failing, can take a look? Make sure to run test command in your local

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**